src/pages/member/QuickRegister.js

```diff
@@ -82,36 +109,43 @@ const EventFormContainer = (props) => {
     return (
       // assumes that the event details component that uses this component (QuickRegister) does not allow
       // the user to get to this page if they are already signed up or the event has passed
```

Comment:
So we will allow users to unregister an event in the eventDetails page right? And the users will be allowed to get to this page again after unregistering the event

Reply:
yes, i think so.